python/podio_config_reader.py

```diff
@@ -321,6 +321,8 @@ def __init__(self, yamlfile):
         "exposePODMembers": True,
         # use subfolder when including package header files
         "includeSubfolder": False,
+        # generate the SIO block handlers
```

Comment:
certainly comment and code are out of sync ;-)

Reply:
Ah yes, most definitely... Thanks for checking